src/shared/ui/section-label/section-label.module.scss

```diff
@@ -0,0 +1,28 @@
+.label {
```

Comment:
Why are you changing styles when there is no need?

Reply:
> Why are you changing styles when there is no need?

Since we are using CSS modules, I decided to simplify the name for the styles. Get it back?

Reply:
"root" class name = component name